sia-react-native/App.js

```diff
@@ -1,3 +1,189 @@
+			)
+		}
+	},
+
+	Profile: {
+		screen: Profile,
+		navigationOptions: {
+			tabBarLabel: 'PROFILE',
+			tabBarIcon: ({ tintColor }) => (
+				<Icon3 name="person-outline" color={tintColor} size={24} />
+			)
+		}
+	}
+}, {
+	navigationOptions: ({ navigation }) => {
+		const { routeName } = navigation.state.routes
+		[navigation.state.index];
+		return {
+			headerTitle: routeName
+		};
+	}
+},
+{
+	tabBarOptions: {
+		activeTintColor: '#fb5b5a',
+		inactiveTintColor: 'white',
+		style: {
+			backgroundColor: '#003f5c',
+			borderTopWidth: 0,
+			shadowOffset: { width: 5, height: 3 },
+			shadowColor: 'black',
+			shadowOpacity: 0.5,
+			elevation: 5
+		}
+	}
+});
+
+
+const DashBoardStackNavigator = createStackNavigator({
+	DashboardTabNavigator: DashboardTabNavigator
+},{
+	defaultNavigationOptions: ({navigation})=>{
+		return {
+			headerLeft: <Icon style={{ paddingLeft: 10}}
+						onPress={ () => navigation.openDrawer()}
+						 name="md-menu" 
+						 size={30}/>
+		};
+	}
+});
+
+const AppDrawerNavigator = createDrawerNavigator({
+	Dashboard: {
+		screen: DashboardTabNavigator
+	}
+});
+
+const styles = StyleSheet.create({
+	container: {
+		flex: 1,
+		backgroundColor: '#fff',
+		alignItems: 'center',
+		justifyContent: 'center',
+	},
+});
+
+*/
+
+
+
+/*
+export default createBottomTabNavigator({
+	Home: {
+		screen: Home,
+		navigationOptions: {
+			tabBarLabel: 'HOME',
+			tabBarIcon: ({ tintColor }) => (
+				<Icon2 name="home" color={tintColor} size={24} />
+			)
+		}
+	},
+
+	News: {
+		screen: News,
+		navigationOptions: {
+			tabBarLabel: 'NEWS',
+			tabBarIcon: ({ tintColor }) => (
+				<Icon4 name="news" color={tintColor} size={24} />
+			)
+		}
+	},
+
+	Events: {
+		screen: EventListFront,
+		navigationOptions: {
+			tabBarLabel: 'EVENTS',
+			tabBarIcon: ({ tintColor }) => (
+				<Icon4 name="calendar" color={tintColor} size={24} />
+			)
+		}
+	},
+
+	Notifications: {
+		screen: Notifications,
+		navigationOptions: {
+			tabBarLabel: 'NOTIFICATIONS',
+			tabBarIcon: ({ tintColor }) => (
+				<Icon4 name="notification" color={tintColor} size={24} />
+			)
+		}
+	},
+
+	Profile: {
+		screen: Profile,
+		navigationOptions: {
+			tabBarLabel: 'PROFILE',
+			tabBarIcon: ({ tintColor }) => (
+				<Icon name="md-person" color={tintColor} size={24} />
+			)
+		}
+	}
+}, {
+	tabBarOptions: {
+		//activeTintColor: '#fb5b5a',
+		activeTintColor: '#FF0000',
+		showLabel: false,
+		inactiveTintColor: '#38444C',
+		style: {
+			backgroundColor: '#15202A',
+			borderTopMargin: 5,
+			shadowOffset: { width: 5, height: 3 },
+			shadowColor: 'black',
+			borderTopWidth: 1,
+			borderTopColor: '#38444C',
+			shadowOpacity: 0.5,
+			elevation: 5
+		}
+	}
+})
+
+const styles = StyleSheet.create({
+	container: {
+		flex: 1,
+		backgroundColor: '#fff',
+		alignItems: 'center',
+		justifyContent: 'center',
+	},
+});
+*/
+
+
+
 import React from 'react'
 import { createStore, applyMiddleware } from 'redux'
 import { Provider } from 'react-redux'
```

Comment:
Keep all the imports together. Group these with the ones at the top

Reply:
I didn't want to mix up the imports because I had separate code app.js that used a bottom tab navigator and stack navigator but I just commented that code out since we're solely going to use the switch navigator instead